css/main.css

```diff
@@ -3,3 +3,6 @@
 *:before {
```

Comment:
You don't need this style.
Let's add this code instead:
*,
*:before,
*:after {
  box-sizing: border: box;
}